src/components/custom-field-input-number/custom-field-input-number.md

```diff
@@ -0,0 +1,31 @@
+The Custom Field Input Number component represents the UI for a custom field of type number from Mavenlink's API.
+It has all the properties of the Custom Field Input Text component.
+It also implements a similar validation to the custom field of type number from Mavenlink's API:
+A valid number is either: a) positive whole number; b) negative whole number.
+Otherwise, it is an invalid number and the component is in an invalid state.
+
+```jsx
+<CustomFieldInputNumber
+  id="test-id-1"
+  label="Example 1"
+/>
+```
+----
+##### Disabled:
+```jsx
+<CustomFieldInputNumber
+  disabled
+  id="test-id-2"
+  label="Example 2"
+  value={1337}
+/>
+```
+----
+##### Error:
+```jsx
```

Comment:
Maybe a note here about why this is an error, to bring to the fore the `step` config?

Reply:
The error state explanation is in the file, a bit above. I am not sure whether to keep the description in the same section of the page or separate it out.

Reply:
Oh, I missed that bit! Could we change the wording to something like "a positive or negative number evenly divisible by `step`"?

Reply:
It would be "evenly divisible by 1". Or a multiple of 1. I think both sound kinda weird? It feels like "integer" or "whole integer" is more descriptive.

Reply:
With the default `step` of 1, yes, but we allow `step` to be configured. Either way, this definitely isn't a blocker, so I'll leave it up to you.

Reply:
Oh wow, except I'm totally wrong! Maybe we should allow `step` to be configured, but I see now that it currently isn't. 😅 Again, I'll leave that decision up to you, I'm not sure if we _want_ to support a use case for this input where `step !== 1`.

Reply:
For now, I'm going to leave it as a _no_ for supporting more than our Custom Field API requirements. 

But that might change if we go down the path of a generic _Number_ component -- which I am liking more and more er' day.